File: app/src/main/java/bassamalim/hidaya/database/dbs/AthkarCategoryDB.java
package bassamalim.hidaya.database.dbs;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "athkar_categories")
public class AthkarCategoryDB {

    @PrimaryKey
    @ColumnInfo(name = "category_id")
    private final int category_id;
    @ColumnInfo(name = "category_name")
    private final String category_name;
    @ColumnInfo(name = "category_name_en")
    private final String category_name_en;

    public AthkarCategoryDB(int category_id, String category_name, String category_name_en) {
        this.category_id = category_id;
        this.category_name = category_name;
        this.category_name_en = category_name_en;
    }

    public int getCategory_id() {
        return category_id;
    }

    public String getCategory_name() {
        return category_name;
    }

    public String getCategory_name_en() {
        return category_name_en;
    }
}

File: app/src/main/java/bassamalim/hidaya/database/dbs/AthkarCategoryDB.kt
package bassamalim.hidaya.database.dbs

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "athkar_categories")
class AthkarCategoryDB(
    @field:ColumnInfo(name = "category_id") @field:PrimaryKey private val category_id: Int,
    @field:ColumnInfo(name = "category_name") private val category_name: String?,
    @field:ColumnInfo(name = "category_name_en") private val category_name_en: String?
) {
    fun getCategoryId(): Int {
        return category_id
    }

    fun getCategoryName(): String {
        return category_name!!
    }

    fun getCategoryNameEn(): String {
        return category_name_en!!
    }
}